opensearch_py_ml/ml_commons/ml_common_client.py

```diff
@@ -5,6 +5,9 @@
 # Any modifications Copyright OpenSearch Contributors. See
 # GitHub history for details.
 
+
+from typing import List
+
 from opensearchpy import OpenSearch
 
 from opensearch_py_ml.ml_commons.ml_common_utils import ML_BASE_URI
```

Comment:
I'm not sure for python, but for Java, we generally put util method (usually static method) in util class. But I see we just keep several constants in https://github.com/opensearch-project/opensearch-py-ml/blob/main/opensearch_py_ml/ml_commons/ml_common_utils.py, that seems not necessary to keep this file. How about just move these constants to this `ml_common_client.py` to keep the tightly coupled code closer?

Reply:
Yeah idea is also similar here to keep common method and variables in the util class which can be reused in multiple files. These variables are used also in other files like model_uploader.py